AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1490,8 +1490,6 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
 ! ##.subscribe
 oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.com,belretail.by,thecollegefix.com,femmie.ru,prostofinovo.ru,mauijim.com,netpeak.net,wday.ru,buhgalteria.ru,ellegirl.ru,gameslog.ru,gizmag.com,healthhacks.ru,klerk.ru,menshealth.com,rbc.ru,snce.ru,turizm.ru,wegame.com.ua##.subscribe
 !
-androidgreek.com##.sidebar-container > #custom_html-19
-iz.ru##.news-google
```

Comment:
I guess that these rules was deleted by a mistake, could you please fix it?

Reply:
Oops... I'll make a new pr